scripts/lint

```diff
@@ -9,8 +9,8 @@ export SOURCE_FILES="httpcore tests"
 set -x
 
 ${PREFIX}autoflake --in-place --recursive $SOURCE_FILES
-${PREFIX}isort --project=httpcore $SOURCE_FILES
-${PREFIX}black --exclude 'httpcore/_sync/.*' --exclude 'tests/sync_tests/.*' --target-version=py36 $SOURCE_FILES
+${PREFIX}isort --skip-glob 'httpcore/_sync/*' --skip-glob 'tests/sync_tests/*' --project=httpcore $SOURCE_FILES
+${PREFIX}black --exclude 'httpcore/_sync/.*|tests/sync_tests/.*' --target-version=py36 $SOURCE_FILES
```

Comment:
Don't think the `.*` is necessary but eh 🤷‍♂️. Would be nice if black used globs instead of regexes.